### DISCUSSION TITLE
спасибо за майнер!!!!!

### DISCUSSION THREAD

**Discussion by HXFSQ** [Flowseal/zapret-discord-youtube#721] [General]
сенкс

**Comment by betterdistant** [comment]
Может стоит че нить пояснить? Скрины, доказательства?

**Comment by Flowseal** [comment]
Предостеригаю вас, чтобы вы ничего ещё не скачали:
![image](https://github.com/user-attachments/assets/02e6d853-e84c-4263-a6fd-eacf7d1d4875)

по треду: читайте readme, а описание WinDivert'а у меня уже во снах появляется
